src/Components/Leg/Leg.js

```diff
@@ -35,11 +35,20 @@ export const Leg = (props) => {
             <Text style={styles.dateText}>{endDate}</Text>
             </View>
           </View>
-          <View>
-          </View>
+          
+            <View style={styles.sideBySideContainer}>
+          <TouchableOpacity style={styles.sideBySideButton}>
+            <Text style={styles.buttonText} onPress={() => navigate('Transportation', { leg })}>Transportation</Text>
+          </TouchableOpacity>
+          <TouchableOpacity style={styles.sideBySideButton}>
+            <Text style={styles.buttonText} onPress={() => navigate('AddLodgingInfo')}>Add Lodging</Text>
```

Comment:
@EmilyLalonde I know you're working on this bit, see above comment if you don't already have the userId passing through to the AddLogingInfo component from Leg